README.md

```diff
@@ -77,12 +77,11 @@ Future features:
 
 
 ## ♻️ Usage
-Currently, we support 2 module formats on [NPM](https://npmjs.org/package/bpmn-visualization):
+We support various module formats on [NPM](https://npmjs.org/package/bpmn-visualization):
 - [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE)
 - [ESM](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm)
+- CommonJS
```

Comment:
```suggestion
- [CommonJS](https://www.typescriptlang.org/docs/handbook/2/modules.html#commonjs)
```

Reply:
I am not sure pointing to a typescript example is enough to explain what this is. But as you wish

Reply:
You can add another example 😉 
It's just for the people who don't know what is commonJS (like the other packagings)

Reply:
Ok, I am adding the link, we will review everything in #864